src/atomic.cr

```diff
@@ -7,6 +7,37 @@ require "llvm/enums/atomic"
 # reference types or `Nil`, then only `#compare_and_set`, `#swap`, `#set`,
 # `#lazy_set`, `#get`, and `#lazy_get` are available.
 struct Atomic(T)
+  # Specifies how memory accesses, including non atomic, are to be reordered
+  # around atomics. Follows the C/C++ semantics:
+  # <https://en.cppreference.com/w/c/atomic/memory_order>.
+  #
+  # By default atomics use the sequentially consistent ordering, which has the
+  # strongest guarantees. If all you need is to increment a counter, a relaxed
+  # ordering may be enough. If you need to synchronize access to other memory
+  # (e.g. locks) you may try the acquire/release semantics that may be faster on
+  # some architectures (e.g. X86) but remember that an acquire must be paired
+  # with a release for the ordering to be guaranteed.
+  enum Ordering
+    Relaxed                = LLVM::AtomicOrdering::Monotonic
```

Comment:
To avoid introducing a term that only LLVM uses. C/C++ and Rust use the term "relaxed" and LLVM's own documentation state that "this is memory_order_relaxed in C/C++" :shrug: